README.md

```diff
@@ -171,32 +171,30 @@ Add the following step to your GitHub workflow (in example are used non-default
 ## Features
 ### Built-in
 #### Release Notes Extraction Process
+This feature searches for release notes in the description of GitHub pull requests, making it easier for maintainers to track changes and updates.
+- **Format:** 
+  - The release notes section have to begin with the title `Release Notes:`, followed by the release notes in bullet points. [See Markdown formatting is supported](https://www.markdownguide.org/basic-syntax/#unordered-lists).
```

Comment:
- If I would read this README, there would be on question that would pop out. Does it have to be `Release Notes:`? Can it be `release notes` without `:` or `Release notes:` with not capital first letters. Right under this sentence you use only `Release notes`. I would specify it in one sentence, so the user knows.
- I would rewrite sentence `See Markdown formatting is supported`. Probably the word `See` is extra here.

Reply:
Understood. First part of the convo is resolved. Please react to the second part of this comment.

Reply:
I see, I missed it. Fixed in commit - 1622020e2cc0b20daf5055ef8053bee0a77e5bd6.